docs/content/latest/explore/ysql-language-features/going-beyond-sql/follower-reads-ysql.md

```diff
@@ -27,11 +27,27 @@ showAsideToc: true
+
+The leader lease mechanism guarantees to serve strongly consistent reads where a client can fetch reads directly from the leader, because the leader under lease will have the latest data.
+The interactive animations in this [blog post](https://blog.yugabyte.com/low-latency-reads-in-geo-distributed-sql-with-raft-leader-leases/) explains the performance improvements using leader leases.
+
+## Introduction to Follower reads
 
 YugabyteDB requires reading from the **leader** to read the latest data. However, for applications that don't require the latest data, or are working with unchanging data, the cost of contacting a potentially remote leader to fetch the data may be wasteful. Your application may benefit from better latency by reading from a **replica** that is closer to the client.
 
-Replicas may not be completely up to date with all updates, so this design may respond with stale data. You can specify how much staleness the application can tolerate. When enabled, read-only operations may be handled by the closest replica, instead of having to go to the leader.
+### Scenario where stale reads are not acceptable
+
+Let's say a user starts a donation page to raise money for a personal cause and the target amount must be met by the end of the day. In such a scenario which is time critical, the user will benefit from accessing the most recent donation made on the page and can keep track of the progress. Such real-time applications would require to get the most latest information as soon as it is available. But if there are applications where fetching slightly stale data is okay, then the follower reads serve that purpose.
+
+### Scenario where an application can handle stale reads
+
+Let's say a social media post gets about a million likes and more continuously. For a post with massive likes as this one, slightly stale reads are acceptable and it does not require immediate updates because the number may not really matter to the user reading the post. Such applications don't require to always process requests from the leader. Instead, a slightly older value from the closest replica can achieve improved performance with low latency.
+
+Follower reads are applicable for applications that can tolerate staleness.
+Replicas may not be completely up to date with all updates, so this design may respond with stale data. You can specify how much staleness the application can tolerate. When enabled, read-only operations may be handled by the closest replica, instead of having to go to the leader. The GUC session variables that PostgreSQL supports can be used to enable follower reads.
```

Comment:
Do we need (or want) to link to a reference for GUC variables, or should we just leave it as-is?

```suggestion
Follower reads are applicable for applications that can tolerate staleness. Replicas may not be completely up to date with all updates, so this design may respond with stale data. You can specify how much staleness the application can tolerate. When enabled, read-only operations may be handled by the closest replica, instead of having to go to the leader. The GUC session variables that PostgreSQL supports can be used to enable follower reads.
```